src/findbugs/excludesFilter.xml

```diff
@@ -1,7 +0,0 @@
-<?xml version="1.0"?>
-<FindBugsFilter>
-    <Match>
-       <Class name="org.biouno.unochoice.model.GroovyScript" />
-       <Bug pattern="SE_BAD_FIELD" />
-    </Match>
-</FindBugsFilter>
```

Comment:
Dead code; and no longer needed because recent versions of SpotBugs pass without this exclusion.

Reply:
Tested with `mvn spotbugs:spotbugs spotbugs:gui` from this branch, it passed with no issues.,l